README.md

```diff
@@ -1,6 +1,69 @@
 # curriculum-cr-training-mobile-menu
 
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Project Name
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+> This is a portfoli project to showcse my coding projects and contact information on the web.
+
+## Built With
+
+- Html
+- Css
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- node
+- web broswer
+- code editor
+
+### Setup
+
+```
+  git clone git@github.com:Ginohmk/Ham_menu_code_review-training.git
+```
+
+```
+ cd Ham_menu_code_review-training
+```
+
+### Usage
+
+` open index.html in web broswer`
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] You should add links to your social media handles like LinkedIn, Twitter, and even your GitHub profile link because If anyone like your project and need to connect with you.